ksql-engine/src/main/java/io/confluent/ksql/util/QueryLoggerUtil.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Switch to CCL. Same comment also for FilterNodeTest, ProcessingLogMessageFactory, ProcessingLoggerFactory, and ProcessingLoggerUtil.